content/docs/gto/get-started.md

```diff
@@ -0,0 +1,162 @@
+---
+description:
+  'Learn how you can use GTO to create Artifact Registry in Git repository'
+---
+
+# Get Started
+
+GTO helps you build a Artifact Registry out of your Git repository. It does so
+by creating Git tags of [special format](/doc/gto/user-guide) and managing
+[`artifacts.yaml` metafile](/doc/gto/user-guide). Since committing large files
+to Git is not a good practice, you shouldn't commit binary files to Git.
+Instead, use DVC, Git-lfs, or any method commit pointers to the binary files
+instead.
```

Comment:
1. First, IMHO: yes. You can put everything in a single repo and be OK with that. You can link artifacts from the other repos, e.g. adding `path: https://github.com/iterative/dataset-registery/mnist` to `artifacts.yaml` will register that artifact (although it should be pinned to commit like https://github.com/iterative/dataset-registry/tree/6ac243e9875e5086be1488dacbb56f2c23e44768/mnist, but these are details).
  Second, I get your point about cross-repo Registry and one-repo Registry. If you see a better term, please suggest, otherwise we can just mention it's "single-repo", not "cross-repo".
2. Yes, the example in the link above
3. It's not "machine learning artifact registry" to be precise, so I suggest something like
```
GTO helps you build an Artifact Registry on top of a Git repository (a special case
is Machine Learning Model Registry). You can register relevant versions of your files
(e.g. ML model releases) and assign them to different deployment environments (testing, shadow,
production, etc.). Git-native mechanisms are used, so you can automate the
delivery of your Ml project with CI/CD, and adopt a GitOps approach in general.
```

Reply:
> You can link artifacts from the other repos

OK cool. We should probably use wording that leaves room for this e.g. "build an artifact registry based on one or more Git repositories".

Same for "files and directories", just to be clear.

> (a special case is Machine Learning Model Registry)

Too specific for the intro. Also, I don't understand why artifact registries are not ML but models reg are. But no problem, just avoid "machine learning" here if you prefer.

Reply:
> OK cool. We should probably use wording that leaves room for this e.g. "build an artifact registry based on one or more Git repositories".

There are two things you can think of when you read that phrase. First, it's based on a single Git repo, just because `artifacts.yaml` and git tags should be a part of this one repo. Second, you can reference files/dirs from any folder. Let's better try to address the 2nd with something like `You can register relevant
versions of your files or directories from any source`? To not confuse two ideas.

> I don't understand why artifact registries are not ML but models reg are

I'm trying to convey a different idea: your Artifact registry is general, and could be not-ML-specific at all. I'll remove Model Registry mention in the first paragraph, but will mention we work with MR when we clone `example-gto` repo.

Reply:
> better try to address the 2nd with something like "You can register relevant versions of your files or directories from any source"

"artifact registry based on one or more Git repositories" sounds equivalent to me but if you prefer to use 2 separate sentences, that can work too.

> Artifact registry is general, and could be not-ML-specific

I think it's safe to assume most **ML**EM docs readers are involved in ML, TBH. I think that's our target user?

Reply:
> I think it's safe to assume most MLEM docs readers are involved in ML, TBH. I think that's our target user?

Yes. These docs will be published on dvc.org eventually, btw.

Reply:
> These docs will be published on dvc.org eventually

WDYM? This will go to in mlem.ai/doc/gto right?